layouts/partials/article-meta.html

```diff
@@ -1,6 +1,6 @@
 			<p>
 				{{ if not (eq (.Date.Format "2006-01-02") "0001-01-01") }}<time>{{ .Date.Format "2006-01-02" }}</time>{{ end }}
-{{ if default true (index .Site.Params "show-wordcount") }}
+{{ if default true (index .Site.Params "ShowWordCount") }}
```

Comment:
What about:
```
{{ if default true .Site.Params.showWordCount }}
```
I have been looking around and I have realized that currently all site parameters are all lower case: `.Site.Params.showwordcount`. I think I had some issues when using JSON but we should probably go for camel case (`showWordCount`) and not pascal case (`ShowWordCount`).